### PR TITLE
Update dockerfile for the dotnet build

### PR DESCRIPTION
**Pull request by harshithmohan**
I'm gonna put this here before I accidentally delete these files.
Everything's working properly for me, just check if it's ok and if it is, time for new daily builds?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,54 +1,32 @@
...
 
-RUN apt-get update && apt-get install -y apt-utils libmediainfo0v5 librhash0 sqlite.interop jq unzip libunwind-dev && apt-get install -y -t stretch-backports gosu
+RUN apt-get update && apt-get install -y apt-utils gosu jq unzip libmediainfo0v5 librhash-dev libunwind-dev mono-runtime libmono-system-xml-linq4.0-cil
 
 WORKDIR /usr/src/app/build
 COPY --from=0 /usr/src/app/build .
 COPY ./dockerentry.sh /dockerentry.sh
 
 WORKDIR /usr/src/app/build/webui
-#RUN curl -L $(curl https://api.github.com/repos/ShokoAnime/ShokoServer-WebUI/releases | jq -r '. | map(select(.prerelease==false)) | .[0].assets[0].browser_download_url') -o latest.zip
```
Comment (Cazzar) [inline Dockerfile:51]: Don't delete this line.

File: Dockerfile [hunk 2]
```diff
@@ -1,54 +1,32 @@
-FROM mono:6.0
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
 
 #MAINTAINER Cayde Dixon <me@cazzar.net>
 ENV PUID=1000 \
-    PGID=100  \
-    TargetFrameworkDirectory=/usr/lib/mono/
```
Comment (Cazzar) [inline Dockerfile:6]: Why this directory?

File: Dockerfile [hunk 3]
```diff
@@ -1,54 +1,32 @@
-FROM mono:6.0
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```
Comment (Cazzar) [inline Dockerfile:1]: Any specific reason for mcr.microsoft?

File: Dockerfile (hunk 1 above)
Comment (harshithmohan) [reply Dockerfile:51]: Sorry, I just rewrote the whole file and copy-pasted some things so I think I missed it

File: Dockerfile (hunk 2 above)
Comment (harshithmohan) [reply Dockerfile:6]: As I said, rewrote.
What should that directory point to now? I will fix it

File: Dockerfile (hunk 3 above)
Comment (harshithmohan) [reply Dockerfile:1]: It's the official dotnet core image?
[https://hub.docker.com/_/microsoft-dotnet-core-sdk/](url)

**Pull request merged**